modules/backend/classes/FormField.php

```diff
@@ -4,6 +4,7 @@
 
 use BackedEnum;
 use Html;
+use Lang;
```

Comment:
Can we use the full version of this import instead of the global alias? Would like to gradually migrate that way.

Reply:
done